client/pages/Main.js

```diff
@@ -20,6 +22,8 @@ export default function Main() {
                         <Route path="/books" component={Books} />
                         <Route path="/signup" component={Register} />
                         <Route path="/signin" component={SignIn} />
+                        <Route path="/forgot" component={ForgotPassword} />
```

Comment:
What did the user forget ? Let's make the url descriptive. Let it be `/forgot-password` and `/reset-password`